kubernetes/config-maps/concent-api/local_settings.py.j2

```diff
@@ -7,6 +7,7 @@ from ..secrets.secrets import (
     EMAIL_PASSWORD,
     CONCENT_PUBLIC_KEY,
     CONCENT_PRIVATE_KEY,
+    SENTRY_DNS,
```

Comment:
It's `DSN`, not `DNS`. Maybe use full name to avoid mistakes like this: `SENTRY_DATA_SOURCE_NAME`.

Reply:
Done. Change name to `SENTRY_DATA_SOURCE_NAME`.